taskograph.Helpers/Messages.cs

```diff
@@ -27,5 +27,10 @@ public static string GetErrorMessage(string exceptionMessage)
 
         //Task 
         public const string NULL_VALUE = "N/A";
+
+        //Error messages
+        public const string ERROR_NO_TASKS = "There are no tasks present. Please add at least one task before you continue.";
```

Comment:
Dobra praktyka, tylko widzę że Ci przemykają dodatkowo funkcjonalności do PRa, który miał robić coś innego. Uważaj na to, bo potem chcąc wycofać konkretne zmiany, będziesz je w stanie wycofać tylko grupowo, więc i wiadomości z błędami zostaną usunięte.

Reply:
Tak, ten projekt zastał mi się bardzo i już nie ogarniam co się w nim dzieje. Niech to już pójdzie jak jest a następnie będę się trzymać tematu.